src/server/http/register_hapi_plugins.js

```diff
@@ -20,14 +20,11 @@
 import HapiTemplates from 'vision';
 import HapiStaticFiles from 'inert';
 import HapiProxy from 'h2o2';
-import { fromNode } from 'bluebird';
 
 const plugins = [HapiTemplates, HapiStaticFiles, HapiProxy];
 
 async function registerPlugins(server) {
-  await fromNode(cb => {
-    server.register(plugins, cb);
-  });
+  return server.register(plugins);
```

Comment:
Can we use `return await` here too?

Reply:
Not sure what the benefit of that would be? The caller still has to `await` it as well.

Reply:
Yeah, personally I just always `return await` when that's what I want because I think it makes it a little clearer to readers down the road that `server.register()` is returning a promise that needs to be awaited, and it helps avoid the situation where someone else comes in and tries to wrap this in a `try {} catch {}` and runs into the problem we ran into in other parts of this PR because we weren't using `return await`.

Reply:
Got it, yeah that makes sense. Thanks.